### ISSUE TITLE
White lines between systray icons

### ISSUE THREAD

**Issue by shodanx2** [nitschis/GreyEveTheme#9]
Hello,

This is about the systray icons

Does anyone have a solution ?

I tried [TranslucentTB](https://github.com/TranslucentTB/TranslucentTB) but it doesn't help for that

![image](https://github.com/nitschis/GreyEveTheme/assets/10621885/26d320af-0ce5-4840-8521-451174770cf2)

It's not terrible, but some people I show my desktop to really focus on that ! 

And that turns them off from using greyeve, even though they like the rest of it.

They don't realize it's not a big deal !

**Comment by Lichtenshtein** [comment]
I don't think so... As far as i remember, the registry parameter that is responsible for the color of the text is also responsible for the color of those lines (for some reason incomprehensible to me). As well as for the color of the window frames (also white). If you make the frames dark, then the text color in all windows will become unreadable (dark).

**Comment by shodanx2** [comment]
Maybe we could hack the dll or exe that creates this systray ?

Or, is there a systray alternative like OpenShell and TranslucentTB are for the start menu and taskbar ?

Or maybe capture and override the WM_PAINT API messages ?

**Comment by Lichtenshtein** [comment]
You may want to try out any of [those](https://winclassic.net/thread/1552/colors-metrics-fonts-schemes-collection) classic Windows themes with [this](https://gitlab.com/ftortoriello/WinClassicThemeConfig) theming tool. I also found [this](https://gitlab.com/ftortoriello/WinTbConf), [this](https://github.com/Abdelrhman-AK/WinPaletter), [this](https://github.com/lukeyou05/tacky-borders), and  [this](https://windhawk.net/mods/classic-theme-enable-with-extended-compatibility). Let me know if something of those works for you. I can't use high-contrast themes at the moment. But i'm wondering if you'll be lucky enough to achieve what you want.
